src/main/java/io/github/townyadvanced/townyresources/commands/TownyResourcesAdminCommand.java

```diff
@@ -88,5 +93,17 @@ private void parseReRollCommand(CommandSender sender) {
 		})
 		.sendTo(sender);
 	}
+
+	private void bypassExtractionLimitCommand(CommandSender sender) throws ExecutionException, InterruptedException {
```

Comment:
Is it likely that either of these exceptions will actually get thrown?

Reply:
They can't be thrown, was messing around with async stuff, remove them.